autoscheduler/feedback/views.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework.decorators import api_view, parser_classes
+from rest_framework.parsers import JSONParser
+from rest_framework.response import Response
+from feedback.models import FeedbackFormResponse
+
+@api_view(['POST'])
+@parser_classes([JSONParser])
+def submit_feedback(request):
+    """ Creates a FeedbackFormResponse object corresponding to the sent data """
+    rating = request.data.get('rating')
+    comment = request.data.get('comment')
```

Comment:
Any particular reason you chose to put the information in the request body rather than as query parameters? Don't see why this route couldn't just be used like `/feedback/submit?comment="asdfassadf"&rating=5`

Reply:
Browsers have max URL lengths (generally 2MB which is problematic if you submit feedback close to the length limit) and I also want to avoid any problems with URL encoding/decoding from potentially cropping up